operator/src/main/java/org/keycloak/operator/controllers/KeycloakDeployment.java

```diff
@@ -442,6 +445,7 @@ private List<EnvVar> getEnvVars() {
         // set env vars
         serverConfigSecretsNames = new HashSet<>();
         List<EnvVar> envVars = serverConfig.stream()
+                .sorted((e1, e2) -> e1.getName().compareTo(e2.getName()))
```

Comment:
I don't think we need to sort it here again, do we?

Reply:
Removed. 

Thanks.